protocol/x/rewards/types/params.go

```diff
@@ -10,9 +10,9 @@ import (
 func DefaultParams() Params {
 	return Params{
 		// Corresponds to module account address: dydx16wrau2x4tsg033xfrrdpae6kxfn9kyuerr5jjp
-		TreasuryAccount:  "rewards_treasury",
+		TreasuryAccount:  TreasuryAccountName,
 		Denom:            "dv4tnt",
-		DenomExponent:    -6,
+		DenomExponent:    -18,
```

Comment:
nit: same here, don't we define what the denom exp is for the native staking token? can we use that const?